packages/teams-js/src/public/appWindow.ts

```diff
@@ -36,39 +31,20 @@ export interface IAppWindow {
 export class ChildAppWindow implements IAppWindow {
   /**
    * Send a message to the AppWindow.
+   *
    * @param message - The message to send
-   * @returns Promise that will be fulfilled when the AppWindow posts back a response
-   */
-  public postMessage(message: any): Promise<void>;
-  /**
-   * @deprecated
-   * As of 2.0.0-beta.1, please use {@link ChildAppWindow.postMessage ChildAppWindow.postMessage(message: any): Promise\<void\>} instead.
-   * @param message - The message to send
-   * @param onComplete - The deprecated way of invoking a callback to know if the postMessage has been success/failed.
+   * @param onComplete - The callback to know if the postMessage has been success/failed.
    */
-  public postMessage(message: any, onComplete: (status: boolean, reason?: string) => void): void;
-  public postMessage(message: any, onComplete?: (status: boolean, reason?: string) => void): Promise<void> {
+  public postMessage(message: any, onComplete?: (status: boolean, reason?: string) => void): void {
```

Comment:
should onComplete be a required parameter? It looks like in the old version of this class it was required, so I was curious about whether what you are doing on line 40 (getGenericOnCompleteHandler) was actually changing how this worked?

Reply:
I cannot seem to find a release where it was required. Do you remember what release was it?

Reply:
I was going off of the old signature (line 49 before your change) where onComplete was a required parameter

Reply:
This was optional in v1, like we talked, we are going to keep it optional going forward.